syntax = "proto3";

package roadwork.system;

message BaseResponse {}